EIPS/eip-1.md

```diff
@@ -42,21 +42,27 @@ The EIP process begins with a new idea for Ethereum. It is highly recommended th
 
-Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the EIP to be mature and ready to proceed to the next phase.
+To allow EIP PR authors time to complete editing, PR titles starting with '[WIP]' will be held off from merging until their first draft is complete when the author can request for their PR to be merged. This prevents merging an EIP when more edits are intended to be made by the author, although it can always be edited after merging.
+
+Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the EIP to be mature and ready to proceed to the next phase. An automated process will check all pull requests; if the request passes the format checks and contains only additions or edits to drafts owned by the submitter, it will be automatically merged. No editorial control is exercised over the content of an EIP draft.
 
 Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track Core EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+Standards track EIPs of type 'Core' are updated from 'Draft' to 'Accepted' status when a core devs meeting has accepted the draft and the participants have announced their intention to implement the EIP for inclusion in a future hard fork. Once a Core EIP has been implemented in at least two clients totalling at least 50% of deployed nodes, and all pass a common set of test suites, the status of the EIP is updated to 'Final'. An EIP can also be assigned status “Deferred”. The EIP author or editor can assign the EIP this status when no progress is being made on the EIP.
+
+Non-core EIPs remain in Draft status until at least one implementation has been completed and referenced in the Implementations section of the draft. At that point, the draft's author may request a 'final call', by submitting a pull request changing the EIP's status from 'Draft' to 'Final'.
```

Comment:
One implementation does not a standard make.

Reply:
A standard isn't made out of implementations at all. How many we require is entirely up to us.

Reply:
I''d rather leave it to the community to decide in the Final Call how much implementation is needed.